react-todo-app/src/TodoContainer/Components/Search/Search.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import style from "./Search.module.css";
+import PropTypes from "prop-types";
+
+const Search = ({ handleSearch }) => {
```

Comment:
Nice job on the search! Since the search is filtering results on the todo section rather than the navbar I'd suggest moving the search feature into the todo section, possibly here: 
![image](https://user-images.githubusercontent.com/6400323/233336438-2dba96bb-aa98-42dd-b8ba-04b6f3563a9d.png)